.github/workflows/e2e.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: 1.22.5
+          go-version: 1.22.3
```

Comment:
why this change?

Reply:
thought this was related to to the docker issue, i can revert back to .5.